tests/test_vqe.py

```diff
@@ -886,6 +887,38 @@ def circuit(params):
         mt2 = circuit.metric_tensor([p])
         assert np.allclose(mt, mt2)
 
+    def test_multiple_devices(self, mocker):
```

Comment:
I suppose, due to the `tape_mode` decorator, this test is being run in both tape and non-tape mode?